src/services/keymanager.ts

```diff
@@ -5,6 +5,7 @@ enum KeyManagerLevel {
     PIN = 'PIN',
     FINGERPRINT = 'FINGERPRINT',
     LOCAL = 'LOCAL',
+    LOCALSTORAGE = 'APP.SSO',
```

Comment:
this is not an app specific storage type

I think change to:
BROWSERLOCALSTORAGE = 'BROWSERLOCALSTORAGE'

You could also create one for session storage as well...